teamwork/apps/profiles/tests.py

```diff
@@ -1,3 +1,63 @@
 from django.test import TestCase
+from django.contrib.auth.models import User
+
+from teamwork.apps.profiles.views.BaseView import find_available_username
+from teamwork.apps.profiles.forms import SignUpForm
 
 # Create your tests here.
+class DuplicateSignUpTest(TestCase):
+    """
+    Creates a user and asserts return values are correct from authenticate method
+    """
+    def setUp(self):
+        """
+        Init any variables that are needed for testing
+        """
+        self.user1 = User.objects.create_user('test', 'test@testing.com', 'pw')
+
+    def tearDown(self):
+        """
+        Delete any variables that were created for testing
+        """
+        del self.user1
+
+    def test_next_available_username(self):
```

Comment:
I think we should avoid mixing camelCase and snake_case for method names but its nbd right now

Reply:
Yeah, agreed. We need to go through and cleanup the camelCase to snake_case in my opinion, since that is the more pythonic styling.